CHANGELOG.md

```diff
@@ -3,3 +3,8 @@
 ## 1.0.0
 
 First version with advanced options from Google Maps API documentation
+
+## 2.0.0
```

Comment:
The newest should be at the top, and dated to the next scheduled release.

Reply:
Ok, done